test/e2e/data/infrastructure-docker/main/clusterclass-quick-start-runtimesdk.yaml

```diff
@@ -54,9 +54,9 @@ spec:
   patches:
   - name: test-patch
     external:
-      generatePatchesExtension: generate-patches.${EXTENSION_CONFIG_NAME:-"k8s-upgrade-with-runtimesdk"}
-      validateTopologyExtension: validate-topology.${EXTENSION_CONFIG_NAME:-"k8s-upgrade-with-runtimesdk"}
-      discoverVariablesExtension: discover-variables.${EXTENSION_CONFIG_NAME:-"k8s-upgrade-with-runtimesdk"}
+      generatePatchesExtension: generate-patches.${EXTENSION_CONFIG_NAME:-k8s-upgrade-with-runtimesdk}
```

Comment:
The quotes are there after envsubst this never worked.. (we just never relied on the default values)